rust_icu_sys/Cargo.toml

```diff
@@ -23,3 +23,15 @@ paste = "0.1.5"
 # Indented documentation text in the generated library is prose, not rust code.
 # See https://github.com/rust-lang/rust-bindgen/issues/378
 doctest = false
+
+[features]
+default = []
+
+# If set, cargo will not run bindgen, but will use the provided static libraries
+# instead.
+disable_bindgen = []
+
+# If set, ICU library version renaming is *not* performed.
+# See http://userguide.icu-project.org/design for a discussion of renaming.
+disable_renaming = []
```

Comment:
Might it be cleaner to make the feature names positive, and have `default = ["bindgen", "renaming"]`?

Reply:
> Might it be cleaner to make the feature names positive, and have `default = ["bindgen", "renaming"]`?

Agreed, except it seems that features are additive. Am I missing something?

Reply:
Users can put
```toml
rust_icu = { default-features = false }
```
in their Cargo.toml files, right?

Reply:
I thought that would have been weird where we plan to use it.  But apparently it isn't.  Changed.